backend/services/inventory/api/http/api_inventory.go

```diff
@@ -403,155 +271,193 @@ func (i *inventoryHandlers) GetDevicesHandler(w rest.ResponseWriter, r *rest.Req
 		HasGroup:  hasGroup,
 		GroupName: groupName}
 
-	devs, totalCount, err := i.inventory.ListDevices(ctx, ld)
+	devs, totalCount, err := i.App.ListDevices(ctx, ld)
 
 	if err != nil {
-		u.RestErrWithLogInternal(w, r, l, err)
+		rest.RenderError(c,
+			http.StatusInternalServerError,
+			errors.New("internal error"),
+		)
```

Comment:
Can you please push the underlying error on the stack here?
```suggestion
		c.Error(err)
		rest.RenderError(c,
			http.StatusInternalServerError,
			errors.New("internal error"),
		)
```

---

Alternatively, you could extend the `rest.utils` package with a new function that does this:
```go
// Same as RenderError but pushed `err` on the stack and outputs `apiMessage` to the caller.
func RenderErrorWithMessage(c *gin.Context, code int, err error, apiMessage string) {
	ctx := c.Request.Context()
	_ = c.Error(err)
	err = &Error{
		Err:       apiMessage,
		RequestID: requestid.FromContext(ctx),
	}
	c.JSON(code, err)
}
```
and then do:
```suggestion
		rest.RenderErrorWithMessage(c,
			http.StatusInternalServerError,
			err,
			"internal error",
		)
```
Which will render a nicer accesslog entry.